homeassistant/components/light/blinksticklight.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
Ah yes that is correct. Python is not that good about it. Never mind this comment